common/eth2_network_config/Cargo.toml

```diff
@@ -18,4 +18,4 @@ serde_yaml = "0.8.13"
 types = { path = "../../consensus/types"}
 eth2_ssz = "0.4.1"
 eth2_config = { path = "../eth2_config"}
-enr = { version = "0.6.2", features = ["ed25519", "k256"] }
+discv5 = "0.2.2"
```

Comment:
I think we don't publish this crate so this is probably a nice improvement

Reply:
Yeah. It also has to match the discv5 version we use in the network crate. So I think its easier keeping track of just one crate version rather than differing crates (i.e discv5::enr vs Enr::enr)